main.go

```diff
@@ -78,7 +78,8 @@ func runMog(opts options) error {
 	}
 
 	if len(cfg.Structs) == 0 {
-		return fmt.Errorf("no source structs found in %v", opts.source)
```

Comment:
Can you adjust the e2e test to add a case that would trigger this?

Reply:
i added one

Reply:
Thank you!!